### ISSUE TITLE
Swagger implementation

### ISSUE THREAD

**Issue by emirhandogandemir** [emirhandogandemir/cloud-infra-generator-api#4]

**Comment by wralith** [comment]
I guess it would be better to work on swagger after implementing dtos since some returned models directly dependent to external libraries. I will do this task happily after that 🖖

**Comment by emirhandogandemir** [comment]
> I guess it would be better to work on swagger after implementing dtos since some returned models directly dependent to external libraries. I will do this task happily after that 🖖

You're right. I will inform you after handling the DTOs. Thank you in advance for your support.
